sources/api/schnauzer/src/helpers.rs

```diff
@@ -307,6 +307,9 @@ mod error {
             number: usize,
             source: std::num::TryFromIntError,
         },
+
+        #[snafu(display("Variant {} is invalid, use docker or k8s", var))]
+        InvalidVariant { var: String },
```

Comment:
I would avoid using the term "Variant" since it's overloaded in the Bottlerocket context 😀 
```suggestion
        InvalidOutputType { var: String },
```